README.md

```diff
@@ -1,34 +1,98 @@
 # Red Hat Developers Site
 
-## Getting Started
-This section covers the steps you need to do in order to setup your environment and get the site running for the first time. Further sections cover the details.
+Powering the Red Hat Developers site available at http://developers.redhat.com/
 
+## Developer setup
+We recommend that Docker be used in development. This simplifies the setup, and makes your development environment consistent with other developers and our CI servers. The following sections cover the steps you need to do in order to setup your environment and get the site running for the first time.
+
+Skip to the [Site Build Setup](#site_build_setup) section if you don't want to use Docker.
+###Utilities
+You should be running a bash shell (Linux or OSX) and you will require: git and curl.
+###Docker and Docker compose
+Follow the instructions to install the latest docker for your system [here] (https://docs.docker.com/installation/). It's unlikely you will want to use the packages provided by your system as they will be too far out of date.
+
+Follow the instructions to install the latest docker-compose [here] (https://github.com/docker/compose/releases)
+### Basic Ruby install
+In this project docker and docker-compose are managed through the ruby script found at `_docker/control.rb`. In order to run this you will require ruby 2.1.2. The following instructions install ruby via rbenv. You can use other methods, but your mileage may vary.
```

Comment:
Ruby 2.1.2 should be the base line, but it runs fine on any 2.1 or higher version of ruby.